aiosmtpd/tests/test_smtp.py

```diff
@@ -29,8 +89,17 @@ def factory(self):
 
 
 class TimeoutController(Controller):
+    Delay: float = 2.0
+
+    def factory(self):
+        return Server(self.handler, timeout=self.Delay)
+
+
+class RequiredAuthDecodingController(Controller):
     def factory(self):
-        return Server(self.handler, timeout=0.1)
```

Comment:
Original timeout of `0.1` is too fast for Windows, causing intermittent fails.

By making this into a class attribute it allows easier tuning because changing this automatically adapts the failure case in `TestTimeout.test_timeout`